### ISSUE TITLE
[REQ] p2pbg.com

### ISSUE THREAD

**Issue by lapd87** [Jackett/Jackett#4888]
Can you add p2pbg.com tracker to your list.
Note though that it has Threat: JS/CoinMiner.AH potentially unwanted application which is mining while you are logged in the site.
I have safely used it but for just logging searching download and closing it. This way the mining process is just under a minute and I don't really mind since they have to support the site.
I will try to make mapping and post it here if you can confirm the above thread is not a problem.
Thank you!

**Comment by garfield69** [comment]
try out jackett pre-release v0.11.56

**Comment by lapd87** [comment]
you saved the day again!
thank you!

**Comment by garfield69** [comment]
http://p2pbg.com domain expired 14/apr/2019
is there a new domain or signs of renewal for the old one?

**Comment by lapd87** [comment]
Hi,
I saw it yesterday too. Don't know about new domain. Will search the forums
though. I believe there is some grace period before the domain becomes
publicly available for purchase?

**Comment by lapd87** [comment]
Hi again,
there is no chat in the local forums about the tracker. I am not sure what's going on... I will keep you posted if I find any info.

**Comment by lapd87** [comment]
Hi,
the tracker is back online.

**Comment by lapd87** [comment]
Hi there,
just saw a bug in duckieTV and wanted to let you know :)
When you start torrent download from the app and then close it, after the
download completes but is not updated because duckie is not running and
then you delete it from the torrent app the bar below episode name stays
thicker. I would suggest removing it if duckie can not find the download in
the torrent app or after marking the episode as watched. Currently the
workaround is to unmark as downloaded and mark as downloaded/seen again.

Thank you for the good work and hope you are safe and healthy in this
troublesome times!

На сб, 20.04.2019 г. в 8:53 mario87lapd . <mario87lapd@gmail.com> написа:

> Hi,
> I saw it yesterday too. Don't know about new domain. Will search the
> forums though. I believe there is some grace period before the domain
> becomes publicly available for purchase?
>

**Comment by garfield69** [comment]
@lapd87  are you reporting a bug for duckietv? then you need to open a ticket on https://github.com/SchizoDuckie/DuckieTV/issues/new
this is the repo for the Jackett app.

**Comment by lapd87** [comment]
Sorry for the mistake and thank you !

На чт, 7.05.2020 г. в 9:08 garfield69 <notifications@github.com> написа:

**Comment by garfield69** [comment]
http://p2pbg.com/ returning ERR_CONNECTION_TIMED_OUT

**Comment by lapd87** [comment]
yeah no info why but yesterday it was working

**Comment by ilike2burnthing** [comment]
back up

**Comment by lapd87** [comment]
working now :)
http://alein.org is not though

**Comment by garfield69** [comment]
http://p2pbg.com/ returning 502 Bad Gateway nginx

**Comment by lapd87** [comment]
Yes from late sunday... I believe they just have bad support because it has
happened before to not work for some periods and come back.

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
http://p2pbg.com/ returning 502 Bad Gateway nginx

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
http://p2pbg.com/ returning 502 Bad Gateway nginx

**Comment by ilike2burnthing** [comment]
back up

**Comment by ilike2burnthing** [comment]
Actually, is it just for me (brand new account) or is the only working category TV BG?

**Comment by garfield69** [comment]
you are correct. on the website the torrent search engine is ignoring any requests for a specific category and returning only tv-bg results.

**Comment by ilike2burnthing** [comment]
Search is essentially broken unless using Bulgarian IP - http://p2pbg.com/smf/index.php?topic=3572.msg295684#msg295684

Seems like a bit of a weird region lock, but ok.

**Comment by garfield69** [comment]
http://p2pbg.com/ returns 502 Bad Gateway

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
http://p2pbg.com/ ERR_SQL_ERRin /var/www/nginx/joro/include/functions.php, line 354

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
http://p2pbg.com/
- ERR_CONNECTION_TIMED_OUT
- ERR_CONNECTION_REFUSED

uptime all CP failed

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
http://p2pbg.com/
- ERR_CONNECTION_TIMED_OUT
- GoDaddy domain search page

**Comment by garfield69** [comment]
online

**Comment by ilike2burnthing** [comment]
Annoyingly still redirects for me, but Uptrends shows only 3 checkpoints down, so much be a limited issue.

**Comment by garfield69** [comment]
http://p2pbg.com/ 502 Bad Gateway nginx
all uptime checkpoints return HTTP Protocol Error

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
http://p2pbg.com/
- 502 Bad Gateway nginx
- HTTP ERROR 500

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
http://www.p2pbg.com/ ERR_CONNECTION_TIMED_OUT

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
- http://www.p2pbg.com/ No such file or directory 
Just me? a regional block perhaps?
- now 502 Bad Gateway nginx

**Comment by lapd87** [comment]
Locally is the same :) i suspect it will be back in few days as before

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
- http://www.p2pbg.com/ 502 Bad Gateway nginx
- Table './joro/zsmf_members' is marked as crashed and last (automatic?) repair failed

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
http://www.p2pbg.com/ 502 Bad Gateway nginx

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/4888#issuecomment-2263566148

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/4888#issuecomment-2263566148

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
https://github.com/Jackett/Jackett/issues/4888#issuecomment-2263566148

**Comment by garfield69** [comment]
online

**Comment by garfield69** [comment]
- http://www.p2pbg.com/index.php?page=apples > Temporary unavailable.
- http://www.p2pbg.com/ > 403 Forbidden nginx
- https://www.p2pbg.com/ > TCP Connection Error

**Comment by handokota** [comment]
Been down for a month, should be removed.

**Comment by garfield69** [comment]
>Been down for a month, should be removed.

Thanks, but there is no need to post reminders, we monitor indexer status daily. 
We allow 30 days from the time we open the comment that the site is offline, which was Feb 2nd. 
So if the site is still offline after Mar 2nd we will remove the indexer.

**Comment by garfield69** [comment]
online, but indexer needs updating

**Comment by garfield69** [comment]
v0.24.1234

**Comment by garfield69** [comment]
https://www.p2pbg.com/ > Maintenance work. We will be back with many new things soon!

Login page back up, but trying to login either returns `419 | Page Expired` or redirect to https://www.mar0der.com/login - can login there, but the logo and recent announcements show p2pbg.com, and some URLs on the site are for `http://localhost:4080/...`, so I'm guessing it's a temp site.

**Comment by garfield69** [comment]
online, however login form and browse page have been changed, I will update the indexer.

**Comment by garfield69** [comment]
Hmm. the site may be moving to PAY2DL. Magnets currently do not show up hardcoded into the results like previously, but have 
`Magnet links require Super User or VIP. Open the marketplace?`
However https://www.p2pbg.com/marketplace link takes you to a `Maintenance work` page
I'll make magnets optional for now.

**Comment by garfield69** [comment]
https://www.p2pbg.com/ > Maintenance work We will be back with many new things soon!

**Comment by garfield69** [comment]
back online, indexer update continues

**Comment by garfield69** [comment]
v0.24.1459

**Comment by garfield69** [comment]
https://www.p2pbg.com/ > 504 Gateway Time-out nginx/1.29.4

**Comment by garfield69** [comment]
online
